content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -510,116 +524,56 @@ The SNI proxy will forward the traffic to port `443`.
     metadata:
       name: forward-downstream-sni
     spec:
-      filters:
-      - listenerMatch:
-          portNumber: 443
-          listenerType: SIDECAR_OUTBOUND
-        filterName: forward_downstream_sni
-        filterType: NETWORK
-        filterConfig: {}
+      configPatches:
+      - applyTo: NETWORK_FILTER
+        match:
+          context: SIDECAR_OUTBOUND
+          listener:
+            portNumber: 443
+            filterChain:
+              filter:
+                name: mixer
```

Comment:
Must be `istio.stats` instead of mixer, it seems the `mixer` filter was replaced by `istio.stats`.